source/README.md

```diff
@@ -0,0 +1,59 @@
+# Core Specification Support
+
+- [MaterialXCore](MaterialXCore): Support for the core MaterialX elements
+and graph traversal.
+- [MaterialXFormat](MaterialXFormat): XML serialization support.
+- [PyMaterialX](PyMaterialX/README.md) : Core library Python API support
+
+# Shader Generation Support
+
+## Supported APIs
+-   C++
+-   Python
+
+## Supported Backends
+
+-   Core shader generation API: [MaterialXGenShader](MaterialXGenShader) module
+    -  Refer to document in module folder.
+-   Shader Generation languages supported:
+    -   GLSL
+        -   Version 4 or higher
+        -   Core support in [MaterialXGenGLSL](MaterialXGenGLSL) module
+    -   OSL:
+        -   [MaterialXGenOsl](MaterialXGenOsl) module.
+        -   Uses an implementation which differs from the reference OSL
+            implementation
+
+## Definition Libraries
+
+-  Standard library ([stdlib](../documents/Libraries/stdlib)) implementations for supported languages.
+-  PBR library ([pbrlib](../documents/Libraries/pbrlib)): node definitions and implementations for supported languages.
+-   Basic "default" non-LUT color management transforms for OSL and
+    GLSL:
+    -   lin_rec709, gamma18, gamma22, gamm24, acescg, srgb_texture
+-   Basic "light" definitions and implementations for GLSL:
+    -   point, directional, spotlight, IBL
+
+## Library Tree Structure
+- Refer to documentation in the [Libraries](../documents/Libraries) folder.
+
+## Unsupported definitions for Shader Generation
```

Comment:
Some of the details in this document seem a bit too low-level for a README at source root.  Perhaps we should restrict this document to a brief overview of the source modules (MaterialXCore, MaterialXFormat, MaterialXShaderGen, etc), and point the developer to other documents (e.g. ShaderGeneration.md, Libraries/README.md) for further details.

Reply:
I wonder if it's better to move the details of this into the DeveloperGuide/ShaderGeneratoin.md file.
How does that sound?

Reply:
That sounds wise to me, and some of this information may already be there.